src/packages/console/src/pages/accountPage.tsx

```diff
@@ -236,112 +229,118 @@ export const AccountPage = (props: IAccountPageProps): React.ReactElement => {
   };
 
   return (
-    <div className={classes.root}>
-      <Elements stripe={stripePromise}>
-        <NavigationBar />
-        <Helmet>
-          <title>{`${account ? account.name : 'Account page'} | Everypage Console`}</title>
-        </Helmet>
-        <main className={classes.content}>
-          <Container maxWidth='lg'>
-            {account === undefined || accountSites === undefined ? (
-              <Typography component='p'>loading...</Typography>
-            ) : account === null || accountSites === null ? (
-              <Typography component='p'>An error occurred. Please try again later.</Typography>
-            ) : (
-              <React.Fragment>
-                <Paper elevation={0} className={classes.paper}>
-                  <Typography variant='h5' className={classes.accountName}>{account.name}</Typography>
-                  <br />
-                  <Typography>
+    <Elements stripe={stripePromise}>
+      <NavigationBar />
+      <Helmet>
+        <title>{`${account ? account.name : 'Account page'} | Everypage Console`}</title>
+      </Helmet>
+      <ResponsiveContainingView size={12}>
+        <Stack direction={Direction.Vertical} isFullWidth={true} paddingTop={PaddingSize.Wide4} paddingBottom={PaddingSize.Wide2} isScrollableHorizontally={false}>
+          {account === undefined || accountSites === undefined ? (
+            <Text tag='p'>loading...</Text>
+          ) : account === null || accountSites === null ? (
+            <Text tag='p'>An error occurred. Please try again later.</Text>
+          ) : (
+            <React.Fragment>
+              <Box variant='card'>
+                <Stack direction={Direction.Vertical} shouldAddGutters={true} defaultGutter={PaddingSize.Default}>
+                  <Text variant='header5'>{account.name}</Text>
+                  <Text>
                     {' You are currently on the '}
-                    <b>{currentPlan.name}</b>
+                    <Text variant='bold'>{currentPlan.name}</Text>
                     {' plan'}
-                  </Typography>
+                  </Text>
                   {nextPlan && (
-                    <Typography variant='caption'>
-                      <Button color='primary' size='small' onClick={(): void => onChangePlanClicked(nextPlan)}>Upgrade</Button>
-                      {' to '}
-                      <b>{nextPlan.name}</b>
-                      {` to ${nextPlan.highlightFeature} and more 🚀`}
-                    </Typography>
+                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Default}>
+                      <Button variant='small-secondary' onClicked={(): void => onChangePlanClicked(nextPlan)} text='Upgrade' />
```

Comment:
i think `variant='small'` will look nicer (without the border)

Reply:
for the button or text

Reply:
button